TLSharp.Core/Session.cs

```diff
@@ -14,9 +14,18 @@ public interface ISessionStore
 
     public class FileSessionStore : ISessionStore
     {
+        private readonly string _storePath;
+
+        public FileSessionStore(string storePath = "Sessions")
```

Comment:
you're providing a default value here which is different than what we had before

Reply:
The default value added to backward compatibility to previous versions.

Reply:
I'm saying that this is not backwards compatible, the "Sessions" string was not there before.

Reply:
@knocte Now it's compatible.